shoppinglist.sublime-project

```diff
@@ -0,0 +1,8 @@
+{
+	"folders":
+	[
+		{
+			"path": "."
+		}
+	]
+}
```

Comment:
This and the file below it should not be in git and should be `.gitignore`'d, especially since they are not useful to people cloning the repo, and `shoppinglist.sublime-workspace` contains information about your personal filesystem ( though nothing important )